src/index.html

```diff
@@ -3,11 +3,297 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
According to the checklist you need to add favicon to the page :)

- [ ] Add favicon